auth-web/src/components/auth/common/Product.vue

```diff
@@ -157,6 +157,11 @@
                 class="mb-0"
                 v-html="$t(productLabel.details)"
               />
+              <p
+                v-if="$te(productLabel.note)"
+                class="mb-0"
+                v-html="$t(productLabel.note)"
```

Comment:
please use `v-sanitize`

Reply:
Can do, would you like this for the previously existing v-html's here, as well?

Reply:
yup, sounds good

Reply:
Updated!
One thing to note for future reference, classes assigned in the template string don't translate through the sanitize. Simple thing to change to a bold tag vs a class, just something noticed 👍 

<img width="869" alt="Screenshot 2024-02-22 at 9 10 04 AM" src="https://github.com/bcgov/sbc-auth/assets/53542131/948c3c67-819c-4b58-8f15-a394a953d501">